src/SmartComponents/Applications/Applications.js

```diff
@@ -1,29 +1,83 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { connect } from 'react-redux';
+import { Skeleton, PageHeader, PageHeaderTitle, Main } from '@redhat-cloud-services/frontend-components';
+import FormRender from '@data-driven-forms/react-form-renderer';
+import { formFieldsMapper, layoutMapper } from '@data-driven-forms/pf4-component-mapper';
+import registryDecorator from '@redhat-cloud-services/frontend-components-utilities/files/Registry';
+import { register } from '../../store';
+import reducers  from '../../store/reduces.js';
+import { getSchema, saveValues } from '../../actions';
 
+@registryDecorator()
 class Applications extends Component {
+    constructor(props) {
+        super(props);
+        register(reducers);
+        props.getSchema(props.match.params.id);
+        this.state = {
+            appName: this.props.match.params.id
+        };
+    }
 
-    render() {
-        const appName = this.props.match.params.id;
+    renderForms(schemas, appName) {
+        return schemas.map((schema, i) => (
+            <FormRender
+                formFieldsMapper={ formFieldsMapper }
+                layoutMapper={ layoutMapper }
+                schema={ schema }
+                onSubmit={ (values) => this.props.saveValues(appName, values) }
+                onCancel={ () => ({}) }
```

Comment:
Can you also implement this function? It should load original values this can be done by calling `this.forceUpdate();`

Reply:
Mhh, but the forms has `canReset`, can we remove the cancel?